vm/src/stdlib/re.rs

```diff
@@ -77,9 +77,8 @@ impl PyValue for PyPattern {
 /// Inner data for a match object.
 #[pyclass(name = "Match")]
 struct PyMatch {
-    start: usize,
-    end: usize,
-    // m: Match<'t>,
+    haystack: PyStringRef,
+    captures: Vec<Option<(usize, usize)>>,
```

Comment:
`Option<(usize, usize)` should be a seperate struct, `Capture` for example, to keep thing simple.